mosquitto/build.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
Pass through [`shellcheck`](https://www.shellcheck.net/).

Use the model of the [Python `build.sh`](https://github.com/unikraft/static-pie-apps/pull/10) for messages, sequencing.